utils/utils.js

```diff
@@ -0,0 +1,73 @@
+const shaLib = require('./sha256.js')
+const ripemd160Lib = require('./ripemd160.js')
+
+module.exports = {
+  serializeHex: (uint8arr) => {
+    if (!uint8arr) {
+      return ''
+    }
+
+    var hexStr = ''
+    for (var i = 0; i < uint8arr.length; i++) {
+      var hex = (uint8arr[i] & 0xff).toString(16)
+      hex = (hex.length === 1) ? '0' + hex : hex
+      hexStr += hex
+    }
+
+    return `0x${hexStr.toLowerCase()}`
+  },
+
+  deserializeHex: (hexStr) => {
+    if (!hexStr) {
+      return new Uint8Array()
+    }
+
+<<<<<<< HEAD
```

Comment:
Merge conflict :D

Reply:
I just pulled the code a few minutes ago and resolved this